includes/cli/class-co-authors-plus.php

```diff
@@ -294,6 +294,8 @@ private static function migrate_linked_guest_authors() {
 				WP_CLI::warning( sprintf( 'No term found for user %d.', $user_id ) );
 			}
 
+			$author_slug = str_replace( 'cap-', '', $guest_term->slug );
```

Comment:
There is an edge case where a username can start with `cap-` and then its term slugs will start with `cap-cap-`. I know that because this -> https://github.com/Automattic/Co-Authors-Plus/blob/1c29766eeda11276bcf6f20e07171b3efcbc7c03/php/class-coauthors-plus.php#L1355

e.g. "Cap Ri" -> "cap-cap-ri".

I think we need to account for that here, right? And only remove the first occurrence of `cap-`

Reply:
Huh, good catch! Fixed in 17b7d10